contract/.openzeppelin/sepolia.json

```diff
@@ -0,0 +1,261 @@
+{
+  "manifestVersion": "3.2",
+  "proxies": [
+    {
+      "address": "0x45E6B2795bAA58616A9a266156C05a0a1ebF1D59",
+      "txHash": "0xe4503680492e54f9081a15e678fb78db7bf4eb3d99dff4e04de70fee2d69c083",
+      "kind": "transparent"
+    },
+    {
+      "address": "0x1bA3e44a58164De1Aa83Faf766EC5521B75F2989",
+      "txHash": "0x2f1e89a3d1c016874063ac2d12f074fbfeb00627d1726743d216a0ece649463d",
+      "kind": "transparent"
+    }
+  ],
+  "impls": {
+    "5720f0db9c5a0d1d50ab303a5cd9a9ae075df32851c7b5f0885d9962329a0f1a": {
+      "address": "0xA3e7f2E9530256adEf2eadb4fA485f348344A21B",
+      "txHash": "0xf6ff9e896c3aae505118da0fe5dc2f31c53626dde90c9566dc1441181599e3cd",
+      "layout": {
+        "solcVersion": "0.8.24",
+        "storage": [
+          {
+            "label": "numSodas",
+            "offset": 0,
+            "slot": "0",
+            "type": "t_uint256",
+            "contract": "VendingMachineV1",
+            "src": "contracts\\VendingMachineV1.sol:9"
```

Comment:
why there is numSodas in here?

Reply:
This was something i was trying with another contract. I have to remove it.